services/web/server/src/simcore_service_webserver/projects/_states_handlers.py

```diff
@@ -159,6 +159,7 @@ async def open_project(request: web.Request) -> web.Response:
 
         return envelope_json_response(project)
 
+    # MD: this might be causing troubles?
```

Comment:
actually now I wonder also. the project is only deemed closed again if there is a DrectorServiceError, I wonder if there are other problems whether the project is also properly closed again?